src/helpers/cn.ts

```diff
@@ -0,0 +1,8 @@
+import {withNaming} from '@bem-react/classname';
+
+export const NAMESPACE = 'df-';
+
+export const cn = withNaming({e: '__', m: '_'});
+export const block = withNaming({n: NAMESPACE, e: '__', m: '_'});
```

Comment:
`cn` and `block` are similar entities and it would be more clear if they had similar names.
And I am sure `block`-name looks more pretty than `b`-one  to provide values for `className` property:
```ts
<div className={block('label')}>...</div>
```
Let's rename the variables:
`cn` => `CN`
`block` => `dfCN`

Reply:
`cn` => `CN` is forbidden by linter:
```
A function with a name starting with an uppercase letter should only be used as a constructor.
``` 
And I'll do the second case renaming now.

Reply:
But this is a strange request, many gravity-ui packages use the naming I suggested

Reply:
> But this is a strange request, many gravity-ui packages use the naming I suggested

Name of a variable should help to get meaning of the variable.  May be I don't understand the logic, but I think the naming they do is strange.

> `cn` => `CN` is forbidden by linter:

Ok, if you don't have better ideas for the names then I want to suggest to rename them in this way:
- `cn` => `block`
- `block` => `dfBlock`

(and leave `b`-variables for specific components)
Actually, I don't think `block`-word is good in here but it looks less confusing than `cn`+`block`.